### ISSUE TITLE
Critical Turkish translation SQL failure

### ISSUE THREAD

**Issue by thiras** [metabase/metabase#9177]
- Chrome Version 71.0.3578.98 (Official Build) (64-bit)
- Ubuntu 18.04 up-to-date
- Postgres 10
- Metabase v0.31.2
- Docker
- Postgres 10 for Internal Database (Separate server from other postgres)
- All system locales, containers locales are default to en_us, even the docker host

Reproduce steps;

1. Do initial setup of Metabase
2. Change Language from Admin panel to Turkish
3. Logout and Login

Description of the problem;

After changing the language to Turkish and re-login; main page has **only** `ERROR: syntax error at or near "İNNER" Position: 63` error message. Therefore it's impossible to reach any homepage elements. Loses all functionality.

Turkish language has 2 kind of I; one is I and other is İ. Be aware second one has dot on it. As the error dumps at the end provide, the query Metabase trying to execute has at least one Turkish capital i with dots(which is illegal character in the SQL queries as you might know). I've also search in the Turkish translation files to detect if somehow contains that `İNNER` word but no success.

External database has almost no data, but just schemas/tables (A migrated Django db). Inner DB also is just created with the metabase (under single docker-compose)

**Note that** With default English language, my setup works flawless. This problem only occurs with Turkish Language. I also confirm, after changing back to English, issue resolves.

Metabase container error log;

```
12-27 22:05:52 ERROR metabase.middleware :: GET /api/database 500 (5 ms) (3 DB calls).
{:message "ERROR: syntax error at or near \"İNNER\"\n  Position: 63",
 :type org.postgresql.util.PSQLException,
 :stacktrace
 ("org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)"
  "org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)"
  "org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)"
  "org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)"
  "org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)"
  "org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)"
  "org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:118)"
  "com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:353)"
  "clojure.java.jdbc$execute_query_with_params.invokeStatic(jdbc.clj:1002)"
  "clojure.java.jdbc$execute_query_with_params.invoke(jdbc.clj:996)"
  "clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1025)"
  "clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1005)"
  "clojure.java.jdbc$query.invokeStatic(jdbc.clj:1099)"
  "clojure.java.jdbc$query.invoke(jdbc.clj:1056)"
  "toucan.db$query.invokeStatic(db.clj:275)"
  "toucan.db$query.doInvoke(db.clj:271)"
  "clojure.lang.RestFn.invoke(RestFn.java:410)"
  "--> models.user$permissions_set.invokeStatic(user.clj:237)"
  "models.user$permissions_set.invoke(user.clj:229)"
  "middleware$bind_current_user$fn__56179$fn__56182.invoke(middleware.clj:136)"
  "models.interface$current_user_permissions_set.invokeStatic(interface.clj:219)"
  "models.interface$current_user_permissions_set.invoke(interface.clj:219)"
  "models.interface$make_perms_check_fn$_has_perms_QMARK___23020.invoke(interface.clj:234)"
  "models.interface$make_perms_check_fn$_has_perms_QMARK___23020.invoke(interface.clj:232)"
  "models.interface$fn__22970$G__22963__22979.invoke(interface.clj:181)"
  "api.database$dbs_list.invokeStatic(database.clj:138)"
  "api.database$dbs_list.invoke(database.clj:137)"
  "api.database$fn__47920.invokeStatic(database.clj:150)"
  "api.database$fn__47920.invoke(database.clj:143)"
  "middleware$enforce_authentication$fn__56174.invoke(middleware.clj:113)"
  "api.routes$fn__56326.invokeStatic(routes.clj:65)"
  "api.routes$fn__56326.invoke(routes.clj:65)"
  "routes$fn__56409$fn__56410.doInvoke(routes.clj:108)"
  "routes$fn__56409.invokeStatic(routes.clj:103)"
  "routes$fn__56409.invoke(routes.clj:103)"
  "middleware$catch_api_exceptions$fn__56309.invoke(middleware.clj:436)"
  "middleware$log_api_call$fn__56287$fn__56289.invoke(middleware.clj:364)"
  "middleware$log_api_call$fn__56287.invoke(middleware.clj:363)"
  "middleware$add_security_headers$fn__56229.invoke(middleware.clj:252)"
  "core$wrap_streamed_json_response$fn__62673.invoke(core.clj:67)"
  "middleware$bind_current_user$fn__56179.invoke(middleware.clj:137)"
  "middleware$maybe_set_site_url$fn__56239.invoke(middleware.clj:290)"
  "middleware$add_content_type$fn__56232.invoke(middleware.clj:262)"),
 :sql-exception-chain ["PSQLException:" "Message: ERROR: syntax error at or near \"İNNER\"" "Position: 63" "SQLState: 42601" "Error Code: 0"]}
``` 

Metabase inner database container log;
```
2018-12-27 19:24:00.876 UTC [32] ERROR:  syntax error at or near "İNNER" at character 63
2018-12-27 19:24:00.876 UTC [32] STATEMENT:  SELECT "p"."object" FROM "permissions_group_membership" "pgm" İNNER JOIN "permissions_group" "pg" ON "pgm"."group_id" = "pg"."id" İNNER JOIN "permissions" "p" ON "p"."group_id" = "pg"."id" WHERE "pgm"."user_id" = 1
```

**Comment by flamber** [comment]
Hi @thiras 
There's a few issues open already about Turkish locale problem, but I'm not sure which one fits your issue best. If you find another issue that fits, then write a comment here with the #number and close this issue.
https://github.com/metabase/metabase/issues?utf8=%E2%9C%93&q=is%3Aissue+turkish

**Comment by camsaul** [comment]
This is caused by https://github.com/jkk/honeysql/issues/236

Submitted PR here: https://github.com/jkk/honeysql/issues/237

**Comment by camsaul** [comment]
This will be fixed once we merge https://github.com/metabase/toucan/pull/59

@walterl since this is the same underlying issue as the other Turkish issue I'm going to assign this to you and you can close them both out once we bump the Toucan and `clojure.java.jdbc` dep versions

**Comment by walterl** [comment]
@camsaul Seeing as this specific case is related to HoneySQL's use of `upper-case` (which you got fixed), shall I update our dep to 0.9.5 and close this? I did not update the HoneySQL dep as part of #9011.

**Comment by camsaul** [comment]
@walterl go for it 👍 Just make sure you PR against `release-0.33.x`
